scripts/enforce-single-runtime.sh

```
@@ -0,0 +1,114 @@
+fi
+
+normalize_interpreter() {
+  local shebang="$1"
+  local trimmed interpreter
+
+  trimmed="${shebang#\#!}"
+  trimmed="${trimmed#${trimmed%%[![:space:]]*}}"
+
+  if [[ "$trimmed" == /usr/bin/env* ]]; then
+    interpreter="${trimmed#/usr/bin/env}"
+    interpreter="${interpreter#${interpreter%%[![:space:]]*}}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)